bin/mac-setup-postgres.py

```diff
@@ -71,10 +71,25 @@ def link_postgres_if_needed(brewname, force=False):
                        check=True, stdout=subprocess.DEVNULL)
 
 
-def install_postgres(brewname: str) -> None:
-    print(f'Installing {brewname}')
-    subprocess.run([BREW, 'install', brewname], check=True)
-    link_postgres_if_needed(brewname, force=True)
+def install_postgres() -> None:
+    # Install an older formula for postgres that is pinned to call icu4c 73.2 
+    # as that is the latest node@16 supports.
+    print('Downloading postgresql@14 with icu4c.rb 73.2 bindings')
+    subprocess.run(['wget', '-O', 'postgresql@14.rb', 'https://raw.githubusercontent.com/Homebrew/homebrew-core/521c3b3f579cd4df16e0b85b26a49e47d2daf9c6/Formula/p/postgresql@14.rb'], check=True)
+    print('Installing postgresql@14 with icu4c.rb 73.2 bindings')
+    subprocess.run(['BREW', 'install', 'postgresql@14.rb'], check=True)
+    link_postgres_if_needed('postgresql@14', force=True)
+    # Reinstall icu4c 73.2 as it will have got updated to 74.2+ during the 
+    # previous postgresql@14 install.
+    print('Downloading icu4c.rb v73.2')
+    subprocess.run(['wget', '-O', 'icu4c.rb', 'https://raw.githubusercontent.com/Homebrew/homebrew-core/74261226614d00a324f31e2936b88e7b73519942/Formula/i/icu4c.rb'], check=True)
```

Comment:
Can we download these files into /tmp/ or something, so we don't pollute the git workspace?